components/Layout/index.tsx

```diff
@@ -8,9 +8,16 @@ import Footer from './Footer/Footer';
 export interface Props {
   children: React.ReactChild;
   goBackButton?: boolean;
+  noLayout?: boolean;
```

Comment:
Should we just make this a boolean, remove option for undefined type, default the parameter to false.

It looks like we are basically using undefined as false

Reply:
done